netbench/netbench-cli/etc/Dockerfile

```diff
@@ -64,7 +64,23 @@ RUN set -eux; \
     cp target/debug/netbench-cli /app/bin; \
   fi; \
   rm -rf target; \
-  echo "#!/usr/bin/env bash\naws s3 cp s3://\$S3_BUCKET/client.json ./client.json\naws s3 cp s3://\$S3_BUCKET/server.json ./server.json\neval /usr/bin/netbench-cli ./client.json ./server.json \$@ > report.json\naws s3 cp ./report.json s3://\$S3_BUCKET/" > /app/bin/start; 
+  # Bash script to retrieve client and server metrics from S3, and upload generated report to S3
+  echo "#!/usr/bin/env bash\n \
+  set -e\n \
+  aws s3 cp s3://\$S3_BUCKET/client.json /tmp/client.json\n \
+  aws s3 cp s3://\$S3_BUCKET/exportedlogs/\$EXPORT_TASK_ID/ /tmp/ --recursive\n \
```

Comment:
You should be able to combine these into a single `aws s3 sync` call. I would also recommend putting them in a `/tmp/netbench` directory so you can then do a single `aws s3 sync` call when uploading it.